src/V3Const.cpp

```diff
@@ -590,6 +590,86 @@ class ConstVisitor : public AstNVisitor {
         return false;
     }
 
+    // 'out' will have those bits set, which are known to be set
+    void setBits(V3Number& out, AstNode* nodep) {
+        // @wsnyder Do I need some sizedness test here, or are they already well formed?
+        if (const AstConst* const np = VN_CAST_CONST(nodep, Const)) {
+            out.opAssign(np->num());
+        } else if (const AstOr* const np = VN_CAST_CONST(nodep, Or)) {
+            // @wsnyder Do I need to check operands are sized and have the same width?
+            V3Number lb(nodep, np->lhsp()->widthMin());
+            V3Number rb(nodep, np->rhsp()->widthMin());
+            setBits(lb, np->lhsp());
+            setBits(rb, np->rhsp());
+            out.opOr(lb, rb);
+        } else {
+            // Pretend we don't know anything
+            out.setAllBits0();
```

Comment:
We know that the converse didn't prove :)  Ditto other func.

```suggestion
            // We couldn't determine any set bits
            out.setAllBits0();
```